examples/cloud-router-2-aws-connection/outputs.tf

```diff
@@ -1,13 +1,9 @@
-output "module_output" {
+output "aws_connection_id" {
   value = module.cloud_router_aws_connection.primary_connection_id
 }
-output "aws_vpc_id" {
-  value = aws_vpc.example.id
-}
-output "aws_vpn_gateway_id" {
-  value = aws_vpn_gateway.example.id
+output "aws_dx_gateway_id" {
+  value = aws_dx_gateway.aws_gateway.id
 }
 output "aws_interface_id" {
-  value = aws_dx_private_virtual_interface.example.id
+  value = aws_dx_private_virtual_interface.aws_virtual_interface.id
 }
-
```

Comment:
Let's keep the trailing newline. Not sure if this is removing it but just double check.

Reply:
I have added the trailing line.